### ISSUE TITLE
Errors should be given in percentage terms

### ISSUE THREAD

**Issue by pgkirsch** [convexengineering/gpfit#108]
Errors seem to be printed as percentages but actually calculated as absolute values

**Comment by whoburg** [comment]
@pgkirsch it looks like we're taking an `exp` before computing error? I think error should be in log space -- screenshot from the paper below to help with discussion

<img width="1033" alt="Screen Shot 2021-07-28 at 11 53 49 PM" src="https://user-images.githubusercontent.com/6042395/127434064-c57b8e97-4d13-4951-9e85-2ce79dd82114.png">

**Comment by pgkirsch** [comment]
@whoburg Ah that's what I was looking for! (I vaguely remembered this being in the paper but somehow missed it.)

But why be satisfied with approximate error when it's so easy to get the exact error in original space? And if we are going to compute max error we need to compute it anyway...

**Comment by whoburg** [comment]
I agree it's easy to compute what any of these errors are, but there's only one of them that our fitting algorithm is actively minimizing (it's minimizing RMS error on the transformed data. The point of equation (42) is that this minimization should also result in low relative error on w.

**Comment by pgkirsch** [comment]
(Copying over from other discussion because it's easier to find here)

Ah fair point. Do you think any of these solutions would help add sufficient clarity?
1. Make the heading of `verbosity=1` output more explicit, i.e. instead of:
```python
Error
-----
```
it could be
```python
Error (in original space) # or log space
-----
```
2. Expand the `self.error` dict to include both types of error, i.e.
```python
self.error = {
    "transformed": {
        "rms": ...,
    },
    "untransformed": { # naming?
        "rms_rel": ...,
        "rms_abs": ...,
        "max_rel": ...,
        "max_abs": ...,
    }
}
```
But it's kinda cumbersome as a user to be like `f.error["untransformed"]["rms_rel"]`
3. Similar but condensed:
```python
self.error = {
    "rms_log": ...,
    "rms_rel": ...,
    "rms_abs": ...,
    "max_rel": ...,
    "max_abs": ...,
}
```

**Comment by whoburg** [comment]
Could choose one that we care the most about and call it `self.error` (it's a float), and then also provide a more complicated `self.errors` dict as you describe above

**Comment by pgkirsch** [comment]
I like that! My vote is for "rms_rel" (untransformed) if that would be ok with you. I think its the most meaningful error to an engineering-application end user.

I'll add a brief discussion of all this to the documentation.

**Comment by pgkirsch** [comment]
Closed by #105
